pkg/telemetry/phonehome/gatherer.go

```diff
@@ -109,8 +109,8 @@ func (g *gatherer) Start(opts ...telemeter.Option) {
 	g.mu.Lock()
 	defer g.mu.Unlock()
```

Comment:
Can we remove this mutex if it's just to serial signal access which is thread safe